test/controller.test.ts

```diff
@@ -2873,50 +2875,136 @@ describe("Controller", () => {
         expect(mocksendZclFrameToEndpoint).toHaveBeenCalledTimes(1);
     });
 
-    it("Respond to genTime read", async () => {
-        const frame = Zcl.Frame.create(0, 0, true, undefined, 40, 0, 10, [{attrId: 0}, {attrId: 1}, {attrId: 7}, {attrId: 4}], {});
-        await controller.start();
-        await mockAdapterEvents.deviceJoined({networkAddress: 129, ieeeAddr: "0x129"});
-        mocksendZclFrameToEndpoint.mockClear();
-        await mockAdapterEvents.zclPayload({
-            wasBroadcast: false,
-            address: 129,
-            clusterID: frame.cluster.ID,
-            data: frame.toBuffer(),
-            header: frame.header,
-            endpoint: 1,
-            linkquality: 19,
-            groupID: 10,
-        });
+    it.each([
+        {
```

Comment:
Since this test has only 1 parameter, I think it makes more sense to put the values in the test.